utils/index.d.ts

```diff
@@ -1331,6 +1331,50 @@ export declare interface ExecuteActivityContext {
      * Children to show under the activity tree item.
      */
     activityChildren?: ActivityChildItemBase[];
+
+    /**
+     * Activity insights to be shared with Copilot
+     */
+    activityInsights: ActivityInsights;
```

Comment:
Originally had this as `commandInsights`, but prefixing with activity seemed more in line with the convention here